### PR TITLE
R docker

### PR DESCRIPTION
**Pull request by jburel**
This PR adds ability to run R notebooks
I have opted to install master since like this we will 5.4.6 
The latest tagged version of the gateway is at 5.4.3

To test locally, run the following commands:
``docker build -t training-notebooks .``
``docker run -it  -p 8888:8888 training-notebooks``
Then check that the installation works by running the R notebook

cc @dominikl @pwalczysko @will-moore

### CODE REVIEW

**Comment by dominikl** [comment]
That's great, thanks @jburel . Master branch has some more functionality as well!

**Comment by dominikl** [comment]
👍 Tested locally, works!

**Comment by jburel** [comment]
It is also available at https://ome-lochy.openmicroscopy.org/jupyterhub-training
using UoD credentials

File: README.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
 # training-notebooks
 
- A set of Notebooks to demonstrate how to access the images and metadata from OMERO.
+A set of Notebooks to demonstrate how to access the images and metadata from OMERO.
 
- The notebooks in this repository are meant to exemplify how to access data in OMERO.
+To build the image run, in this reposiroty:
```
Comment (mtbc) [inline README.md]: repository

File: README.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+
+To start the image:
+
+    $ docker run -it  -p 8888:8888 training-notebooks
+
+The notebooks in this repository are meant to exemplify how to access data in OMERO.
+
+| **Notebook** | **Lang** | **Description** |
+|--------------|----------|-----------------|
+| **[OMERO example](Python/OMEROHelloWolrdNotebook.ipynb)** | Py | Shows how to connect to OMERO and read data. |
```
Comment (mtbc) [inline README.md]: unfortunate typo in filename

File: README.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+| **[Calculate Sharpness](Python/CalculateSharpness.ipynb)** | Py | Calculates sharpness of images and generates heatmaps. |
+| **[Illumination Correction](Python/CalculateSharpness.ipynb)** | Py | Correct the selected image. |
```
Comment (mtbc) [inline README.md]: same file as above

File: README.md (hunk 2 above)
Comment (jburel) [reply README.md]: I am going to rename the playbook

**Comment by mtbc** [comment]
That looks better, thank you.

**Comment by jburel** [comment]
Installation has been retested. Everything is working
Merging so @manics can deploy on vae

**Pull request merged**